### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by girish54321**
Note added for react-native link.

### CODE REVIEW

**Review by Kladislav — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -32,7 +32,10 @@ Run `npm i react-native-lottie-splash-screen --save`
 
 #### Automatic installation
 
-`react-native link react-native-lottie-splash-screen` or `rnpm link react-native-lottie-splash-screen`
+* NOTE: For react-native version => 0.60 you don't need to run `react-native link` auto linking will do the job just fine.
+
+`react-native
```
Comment (HwangTaehyun) [inline README.md:37]: react-native's link command is missing here.
Could you fix this line with `react-native-lottie-splash-screen` or `rnpm link react-native-lottie-splash-screen`?
Also, It seems one line is better!

Comment (girish54321) [reply README.md:37]: Sure.

Comment (girish54321) [reply README.md]: Hi I made small change in readme you can please check.

Comment (HwangTaehyun) [reply README.md]: I check your new commit. It's good!!
Thank you. I am going to accept this pull request.

Comment (girish54321) [reply README.md]: 👍

**Comment by HwangTaehyun** [comment]
Next time, it will be better to commit like the below format.
`[feat or fix or docs] Commit Message`
Thank you!

File: README.md [hunk 2]
```diff
@@ -26,15 +26,34 @@ Works on IOS and Android.
 
 ### First step(Download):
 
-Run `npm i react-native-lottie-splash-screen --save`
+Run `npm i react-native-lottie-splash-screen --save` or `yarn add react-native-lottie-splash-screen`
 
 ### Second step(Plugin Installation):
 
-#### Automatic installation
+### React Native v0.60+
```
Comment (HwangTaehyun) [inline README.md:33]: Good

**Review by HwangTaehyun — COMMENTED** [review]
Good

**Pull request merged**